packages/components/selectFileUploader/SelectFileUploader.web.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { FC, SyntheticEvent, useRef, useState } from "react";
+import { TouchableOpacity, View } from "react-native";
+
+import { SelectFileUploaderProps } from "./SelectFileUploader.type";
+import { formatFile } from "./formatFile";
+import addSVG from "../../../assets/icons/add-circle.svg";
+import { useFeedbacks } from "../../context/FeedbacksProvider";
+import {
+  gradientColorBlue,
+  gradientColorDarkerBlue,
+  gradientColorTurquoise,
+  neutral17,
+  primaryColor,
+  withAlpha,
+} from "../../utils/style/colors";
+import { fontSemibold14 } from "../../utils/style/fonts";
+import { layout } from "../../utils/style/layout";
+import { BrandText } from "../BrandText";
+import { DeleteButton } from "../FilePreview/DeleteButton";
+import { SVGorImageIcon } from "../SVG/SVGorImageIcon";
+import { Box } from "../boxes/Box";
+import { Label } from "../inputs/TextInputCustom";
+
+export const SelectFileUploader: FC<SelectFileUploaderProps> = ({
+  label,
+  style,
+  onUpload,
+  // multiple is not used at true for now, needs to refactor in parents
+  multiple,
+  mimeTypes,
+  children,
+  maxUpload,
+  isImageCover,
+  fileHeight = 256,
+  setIsLoading,
+}) => {
+  const { setToastError } = useFeedbacks();
+  const hiddenFileInput = useRef<HTMLInputElement>(null);
+  const [file, setFile] = useState("");
```

Comment:
Please use another name, exp: fileStr, fileObjURL ... because we could think that it's file object here